lang/camlp5/Portfile

```diff
@@ -28,12 +28,13 @@ checksums           rmd160  459a703b56523f907f0c0c8b9ebf98f11edad676 \
                     size    1284440
 
 depends_lib       port:ocaml \
-    port:ocaml-bos \
```

Comment:
Maybe if tossing the dependencies, sort them alphabetically?

Reply:
I explicitely sorted them like opam file for maintenance purposes, but let's @pmetzger decide, I can amend this as well.

Reply:
I think the usual current practice is to sort alphabetically, but I have no trouble with you trying to sync it with the opam file. (TBH, it would be nice to have an opam to portfile tool. I wonder how hard that would be to build.)

Reply:
Well, this is not my port, so I leave it to whomever wants to decide on this.
Alphabetic sorting makes maintaining easier btw.

Reply:
Let's sort alphabetically then.